src/installer.redscript.ts

```diff
@@ -72,8 +73,8 @@ export const detectRedscriptToplevelLayout = (fileTree: FileTree): boolean =>
   && dirWithSomeIn(FILETREE_ROOT, matchRedscriptFile, fileTree);
 
 const detectRedscriptLayout = (fileTree: FileTree): boolean =>
-  allRedscriptConfigFiles(fileTree).length > 0
-  || dirWithSomeUnder(FILETREE_ROOT, matchRedscriptFile, fileTree);
+  (allRedscriptConfigFiles(fileTree).length > 0 || dirWithSomeUnder(FILETREE_ROOT, matchRedscriptFile, fileTree))
+  && !detectRed4ExtCanonOnlyLayout(fileTree); // since Red4Ext mods can have embedded reds, we don't want to mistake oe for the other
```

Comment:
Alternative here to just switch the order of these two in the installer pipeline?

Reply:
Ya, I wasn't sure what possible issues that could present, and I just wanted to do something quick for this.